sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py

```diff
@@ -255,18 +272,4 @@ def queue_size(self):
                 :caption: Close down the handler.
 
         """
-        self._running = False
-        if self._error:
-            return
-        if isinstance(exception, errors.LinkRedirect):
-            self._redirected = exception
-        elif isinstance(exception, EventHubError):
-            self._error = exception
-        elif isinstance(exception, (errors.LinkDetach, errors.ConnectionClose)):
-            self._error = ConnectError(str(exception), exception)
-        elif exception:
-            self._error = EventHubError(str(exception))
-        else:
-            self._error = EventHubError("This receive handler is now closed.")
-        if self._handler:
-            await self._handler.close_async()
+        await super(EventHubConsumer, self).close()
```

Comment:
We probably could remove this close, as it's simply calling parent's close.

but not this time, could be in the next release